deployment/k8s/charts/service-manager/templates/sm03-deployment.yaml

```diff
@@ -0,0 +1,44 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: {{ template "service-manager.fullname" . }}
+  labels:
+    app: {{ template "service-manager.name" . }}
+    chart: {{ template "service-manager.chart" . }}
+    release: {{ .Release.Name }}
+    heritage: {{ .Release.Service }}
+spec:
+  replicas: {{ .Values.replicaCount }}
+  selector:
+    matchLabels:
+      app: {{ template "service-manager.name" . }}
+      release: {{ .Release.Name }}
+  template:
+    metadata:
+      labels:
+        app: {{ template "service-manager.name" . }}
+        release: {{ .Release.Name }}
+    spec:
+      volumes:
+      - name: config-volume
+        configMap:
+          name: {{ template "service-manager.fullname" . }}
+      containers:
+      - name: {{ .Chart.Name }}
+        image: "{{ .Values.image.repository }}:{{ .Values.image.tag }}"
+        imagePullPolicy: {{ .Values.image.pullPolicy }}
+        volumeMounts:
+        - name: config-volume
+          mountPath: /etc/config
+          readOnly: true
+        env:
+        - name: SM_DB_URI
+          valueFrom:
+            secretKeyRef:
+              name: {{ template "service-manager.fullname" . }}-secret
+              key: uri
+        args: ["config_location=/etc/config/"]
```

Comment:
you could put the arg value in the values.yaml so it's not hardcoded in here . If you decide to do so, replace the mountPath on line 32 as well